src/components/dismissible-box/dismissible-box.style.ts

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from "styled-components";
+import Box from "../box";
+import StyledIcon from "../icon/icon.style";
+
+export interface StyledDismissibleBoxProps {
```

Comment:
Is this imported anywhere?

Reply:
it should be used in the main component, I just pushed a change for that.